.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - DB=mysqli
 
 before_install:
+  - sudo sh -c "echo 'precedence ::ffff:0:0/96 100' >> /etc/gai.conf"
```

Comment:
What needs to happen is to make whichever network is losing v6 packets in
the middle fix their shit. Not disabling things..